common/utils.js

```diff
@@ -732,6 +732,10 @@
         // takes pathname attribute of window.location object and returns app name
         // path should be a string literal which appears before #catalog id in URL (/chaise/recordset/)
         function appNamefromUrlPathname(path){
+            if (path.includes("deriva-webapps")) {
```

Comment:
The changes you made won't have an affect on this function, so there's no need to include this. 

The places that we call `appNamefromUrlPathname` have really specific use cases. In `modal.js`, vm.params.appname` is only used to check if we are in `recordedit` to show a reload button. In `utils.js`, this function is called as part of the `chaiseDeploymentPath` function. We would not want to match on a deriva-webapps path and incorrectlyget the chaise path.

Reply:
Removed this appName change.